core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcAdminImportTestUtils.java

```diff
@@ -0,0 +1,442 @@
+package com.scalar.db.storage.jdbc;
+
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.io.DataType;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+public class JdbcAdminImportTestUtils {
+  static final List<String> UNSUPPORTED_DATA_TYPES_MYSQL =
+      Arrays.asList(
+          "BIGINT UNSIGNED",
+          "BIT(8)",
+          "DATE",
+          "DATETIME",
+          "DECIMAL(8,2)",
+          "ENUM('a','b')",
+          "SET('a','b')",
+          "GEOMETRY",
+          "JSON", // we remove this for MariaDB because it is an alias of a supported type, LONGTEXT
+          "NUMERIC",
+          "TIME",
+          "TIMESTAMP",
+          "YEAR");
+  static final List<String> UNSUPPORTED_DATA_TYPES_PGSQL =
+      Arrays.asList(
+          "bigserial",
+          "bit(8)",
+          "bit varying(8)",
+          "box",
+          "cidr",
+          "circle",
+          "date",
+          "inet",
+          "interval",
+          "json",
+          "jsonb",
+          "line",
+          "lseg",
+          "macaddr",
+          "macaddr8",
+          "money",
+          "numeric(8,2)",
```

Comment:
`float4` should be added for https://github.com/scalar-labs/scalardb/pull/931/files#diff-33515a82af5b0cb8f8944cc2237473e9eb8df95bca427831ddfdb72212184009R240 ?

Reply:
No, we don't have to add it because the JDBC data type for PostrgreSQL `real` (`float4`) is always `REAL`.